### PR TITLE
Fix documentation

### PR DESCRIPTION
**Pull request by guerreirobeatriz**
#### Types of changes
- [ ] Bug fix (a non-breaking change which fixes an issue)
- [ ] New feature (a non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -18,18 +18,19 @@ Product Summary is an app responsible for summarizing product information (such
...
-| `product-specification-badges` | 
+| Block name     | Description | 
+| -------------- | ----------- | 
+| `product-summary.shelf` | ![https://img.shields.io/badge/-Mandatory-red](https://img.shields.io/badge/-Mandatory-red)   The [entity](https://help.vtex.com/tutorial/creating-data-entity--tutorials_1265) Provides product summary data to other blocks. As a parent block, it also provides the needed structure and context for its children (listed below) to be rendered. 
+| `product-summary-attachment-list` | Renders a list for the product's [attachments](https://help.vtex.com/tutorial/adding-an-attachment--7zHMUpuoQE4cAskqEUWScU) | 
+| `product-summary-brand`         | Renders the product's brand | 
+| `product-summary-buy-button` | Renders the Buy Button | 
+| `product-summary-description` | Renders the product's description | 
+| `product-summary-image` | Renders the product's image | 
+| `product-summary-list` | Renders a list of products in the Product Summary |
```
Comment (victorhmp) [inline docs/README.md]:
```suggestion
| `list-context.product-list` | Renders a list of products in the Product Summary | 
```

Comment (victorhmp) [reply docs/README.md]: This block has a special name since it is a `list-context` one :)

File: CHANGELOG.md [hunk 2]
```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+### Added
+- Add Product Summary List in the Product Summary README
+- Add descriptions for each exported block
```
Comment (kaisermann) [inline CHANGELOG.md]:
```suggestion
- Add Product Summary List in the Product Summary README.
- Add descriptions for each exported block.
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
```
Comment (victorhmp) [inline CHANGELOG.md]:
```suggestion
```

**Pull request merged**